proto/minder/v1/minder.proto

```diff
@@ -1272,6 +1272,11 @@ message RuleType {
                     optional string mode = 3;
                 }
 
+                message ActionsReplaceTagsWithSha {
+                    // List of actions to exclude from the replacement
```

Comment:
Not for this patch, but it would be nice to get good documentation for the rule_type when we get to amending it, e.g. can I exclude only `actions/checkout` from being expanded or can I also exclude `actions/checkout@v3` to still expand `v4`.